LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -316,22 +345,55 @@ public void SimulationBig1(int subjectCount, int moduleCount)
         #region Reports
         void ReportHeader(string actionName)
         {
-            Console.WriteLine($"******{actionName} is started******\n");
+            switch (this.ReportType)
+            {
+                case ReportType.Verbose:
+                case ReportType.Short:
+                    Console.WriteLine($"******{actionName} is started******\n");
+                    break;
+            }
         }
 
         void ReportFooter(string actionName)
         {
-            Console.WriteLine($"******{actionName} is finished******\n");
+            switch (this.ReportType)
+            {
+                case ReportType.Verbose:
+                case ReportType.Short:
+                    Console.WriteLine($"******{actionName} is finished******\n");
+                    break;
+            }
         }
 
         void ReportItem(string itemName, string actionName, int itemIndex)
         {
-            Console.WriteLine($" {actionName} {itemIndex}\n{itemName}");
+            switch (this.ReportType)
```

Comment:
It is an overkill to use a **switch** statement here. Use just the good old **if**

Reply:
Done